src/core/errors.test.tsx

```diff
@@ -0,0 +1,107 @@
+import { render } from '@testing-library/react'
+import { FetchError } from '@grafana/runtime';
+import { act } from 'react-dom/test-utils';
+import { FloatingError, parseErrorMessage } from './errors';
+import { SystemLinkError } from "./types";
+import React from 'react';
+import { errorCodes } from "../datasources/data-frame/constants";
+
+test('renders with error message', () => {
+  const { container } = render(<FloatingError message='error msg'/>)
+
+  expect(container.innerHTML).toBeTruthy() // refact: get by text
```

Comment:
I see some comments here for refactoring . Is this something you are going to do in this PR or another one ?

Reply:
done